JavaKotlinPracticals/app/src/main/java/webservices/JsonParsingAdapter.kt

```diff
@@ -0,0 +1,46 @@
+package webservices
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.example.javakotlinpracticals.databinding.ActivityRetroRecyclerBinding
+
+class JsonParsingAdapter(private var listUsers: List<Data>, private var context: Context) :
+    RecyclerView.Adapter<JsonParsingAdapter.ViewHolder>() {
+
+    private lateinit var binding: ActivityRetroRecyclerBinding
+
+    inner class ViewHolder(val binding: ActivityRetroRecyclerBinding) :
+        RecyclerView.ViewHolder(binding.root) {
```

Comment:
if you are using the binding then why do we need to set separate items, just hold the binding variable in holder class